src/main/java/org/broadinstitute/hellbender/engine/FeatureManager.java

```diff
@@ -292,22 +303,22 @@ public boolean isEmpty() {
         final List<FeatureCodec<? extends Feature, ?>> candidateCodecs = getCandidateCodecsForFile(featureFile);
 
         // If no codecs can handle the file, it's a user error (the user provided a file in an unsupported format)
-        if ( candidateCodecs.size() == 0 ) {
+        if ( candidateCodecs.isEmpty() ) {
             throw new UserException.CouldNotReadInputFile(featureFile, "no suitable codecs found");
         }
         // If multiple codecs can handle the file, it's a configuration error on the part of the codec authors
         else if ( candidateCodecs.size() > 1 ) {
-            StringBuilder multiCodecMatches = new StringBuilder();
+            final StringBuilder multiCodecMatches = new StringBuilder();
```

Comment:
Making tons of unrelated trivial style changes like this really clutters a big pull request like this -- these should be done separately if needed, not embedded within a huge branch.